python/yugabyte_db_thirdparty/builder.py

```diff
@@ -77,6 +79,10 @@
     '-DTHREAD_SANITIZER',
 ]
 
+# We create a file named like this in each dependency's build directory, with all the relevant
+# environment variables that we set.
+DEPENDENCY_ENV_FILE_NAME = 'yb_dependency_env.sh'
```

Comment:
Do we need this extra file?
Would it be easier to write env vars directly to yb_build_with_cmake.sh?

Reply:
We currently write the env var file for both configure- and CMake-based builds, and I think the environment-only file is still useful.